.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,23 +4,63 @@ about: Create a report to help us improve
 labels: 'bug'
 ---
 
+**Bug was detected in one of the following services:**
+<!-- Please choose in which service was the bug. -->
+
+[ ] - tedge-agent
+
+[ ] - tedge-mapper-collectd
+
+[ ] - tedge-mapper-c8y
+
+[ ] - tedge-mapper-az
+
+[ ] - tedge-mapper-aws
+
+[ ] - tedge-agent
+
+[ ] - c8y-configuration plugin
+
+[ ] - c8y-log-plugin
+
+[ ] - c8y-firmware-plugin
+
+[ ] - Not sure
+***
+
+**Bug reproduction rate:**
+<!-- Please choose the bug reproduction rate. (100% is for always). -->
+
+[___] %
+***
```

Comment:
What's the use of these `***` lines? I see that it's the syntax for bold and italics text combined, but I didn't quite get why you've used these for a few specific sections.

Reply:
`***` was also new to me, but it is valid GitHub markdown for a [horizontal rule](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#horizontal-rule)

Reply:
I just pushed a small [change](0bff634ec334b49ab971752a45bceb445adc1689) to use the more common `---` syntax, and added blank lines before and after (as per [best practices](https://www.markdownguide.org/basic-syntax/#horizontal-rule-best-practices))